src/main/java/teammates/storage/entity/FeedbackResponseStatisticsType.java

```diff
@@ -0,0 +1,20 @@
+package teammates.storage.entity;
+
+/**
+ * The time-interval of the a feedback session.
+ */
+public enum FeedbackResponseStatisticsType {
+    // CHECKSTYLE.OFF:JavadocVariable
+    HOUR("HOUR"), MINUTE("MINUTE");
```

Comment:
For enum, you do not need to take in value string and getValue.
Just declare 
```
public enum Type {
  HOUR;
  MINUTE;
}
```
and access via `HOUR.name()` and `MINUTE.name()`